js/libs/form.js

```diff
@@ -610,6 +610,20 @@ Fliplet.Widget.instance('form-builder', function(data) {
           $form.$forceUpdate();
         }, 10);
 
+        function validateCheckboxValue(value) {
+          if (typeof value === 'string') {
```

Comment:
I don't understand why we're checking if the value is a string. We just need to check if it's an array or not.

```suggestion
          if (Array.isArray(value)) {
            value = [value];
          }
```

What's the purpose of splitting by `,` here, based on the recommended flow ([link](https://github.com/Fliplet/fliplet-widget-form-builder/pull/249#discussion_r350711438))?